### DISCUSSION TITLE
meta information in HTML reports : I don't see CI env

### DISCUSSION THREAD

**Discussion by Diplodokus** [badeball/cypress-cucumber-preprocessor#1144] [Q&A]
Hi,

I am using the latest version 20.0.0 with meta information (lib version, node version, os, ci) that is visible in HTML reports. However, in my report, I can see the lib version, node version, and OS, but not the CI. Is this normal?

![image](https://github.com/badeball/cypress-cucumber-preprocessor/assets/112950812/6f39edfc-88c7-486e-815b-0fc53d84a9f1)

Thanks for your job!

**Comment by badeball** [comment]
`@cucumber/ci-environment` is used to detect CI provider. Is yours among the [supported providers](https://github.com/cucumber/ci-environment?tab=readme-ov-file#supported-ci-servers)? If not, then yes, this is normal.

**Reply by Diplodokus** [reply]
OK thx it's normal because I use Concourse
